custom/icds_reports/utils/data_pull_ap_rj.sql

```diff
@@ -0,0 +1,271 @@
+                                       ->  Nested Loop  (cost=6197.83..721050.78 rows=1 width=99)
+                                             ->  Parallel Bitmap Heap Scan on awc_location_102840 awc_location  (cost=6197.27..49534.02 rows=13912 width=150)
+                                                   Recheck Cond: (state_id = 'f98e91aa003accb7b849a0f18ebd7039'::text)
+                                                   Filter: (aggregation_level = 5)
+                                                   ->  Bitmap Index Scan on awc_location_pkey_102840  (cost=0.00..6183.36 rows=58025 width=0)
+                                                         Index Cond: (state_id = 'f98e91aa003accb7b849a0f18ebd7039'::text)
+                                             ->  Index Scan using chm_awc_idx_102648 on child_health_monthly_102648 child_health_monthly  (cost=0.56..48.26 rows=1 width=78)
+                                                   Index Cond: (awc_id = awc_location.doc_id)
+                                                   Filter: ((thr_eligible = 1) AND (awc_location.supervisor_id = supervisor_id) AND (month = ANY ('{2019-04-01,2019-05-01,2019-06-01,2019-07-01,2019-08-01,2019-09-01,2019-10-01,2019-11-01}'::date[])))
+(24 rows)
+ */
+
+
+
+COPY(SELECT
+    state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    awc_site_code,
+
+    CASE WHEN SUM(CASE WHEN month='2019-04-01'  THEN wer_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN nutrition_status_weighed=1 AND month='2019-04-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-04-01'  THEN wer_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0 END as growth_monitoring_April,
+    CASE WHEN SUM(CASE WHEN month='2019-05-01'  THEN wer_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN nutrition_status_weighed=1 AND month='2019-05-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-05-01'  THEN wer_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as growth_monitoring_May,
+    CASE WHEN SUM(CASE WHEN month='2019-06-01'  THEN wer_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN nutrition_status_weighed=1 AND month='2019-06-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-06-01'  THEN wer_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as growth_monitoring_June,
+    CASE WHEN SUM(CASE WHEN month='2019-07-01'  THEN wer_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN nutrition_status_weighed=1 AND month='2019-07-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-07-01'  THEN wer_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as growth_monitoring_July,
+    CASE WHEN SUM(CASE WHEN month='2019-08-01'  THEN wer_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN nutrition_status_weighed=1 AND month='2019-08-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-08-01'  THEN wer_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as growth_monitoring_Aug,
```

Comment:
Is this different than the weighing efficiency that already exists in aggchildhealth (and maybe aggawc) (wer_weighed and wer_eligible columns), the sum should already exist.

Reply:
yeah. Its already there. I have written this because last time query to agg_child was slower than query to child_health